components/Editor/index.tsx

```diff
@@ -317,90 +318,101 @@ const Editor = ({ readOnly = false }: Props) => {
         >
           <div
             className={cn(
-              'w-full md:w-1/2 flex flex-col',
```

Comment:
What is the reason for this change on line 320? It will break the app on mobile screen..